src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,9 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+
+  width: $catalog-width;
+  margin: 0 auto;
```

Comment:
You can't set margin to block, only for elements. You can use mix(set `page` class for body and add class `page__catalog` and move margin there.

 More details in video.
[video, timing 4:15] (https://mate.academy/learn/css-advanced/bem#/video/911)
![Screenshot 2022-07-21 at 19 28 44](https://user-images.githubusercontent.com/17098736/180265806-696f5c6e-b0db-4579-946c-0d4280919cca.png)
(https://mate.academy/learn/css-advanced/bem#/video/911)